templates/export/articles.html

```diff
@@ -41,6 +42,12 @@ <h2>Articles in Export</h2>
                         <td>{{ article.active_reviews|length }}</td>
                         <td>{{ article.completed_reviews|length }}</td>
                         <td><a target="_blank" class="small secondary button" href="{% url 'manage_archive_article' article.pk %}"><span class="fa fa-sticky-note"></span></a></td>
+                        <td>
+                            <form method="POST">
+                                {% csrf_token %}
+                                <button name="archive" value="{{ article.pk }}" class="small warning button">Archive Article</button>
```

Comment:
as per https://github.com/BirkbeckCTP/janeway/pull/2929, we can use the new `ConfirmableForm` to ask for confirmation